spec/models/sitemap_spec.rb

```diff
@@ -0,0 +1,72 @@
+require 'spec_helper'
+
+describe Sitemap do
+  let!(:url) { 'http://www.agency.gov/boring' }
+  let!(:html) { read_fixture_file("/html/page_with_og_metadata.html") }
+  let!(:valid_attributes) { { url: url } }
+  let!(:sitemap) { Sitemap.new(valid_attributes) }
+  let!(:existing_domain) { SearchgovDomain.create!(domain: 'existing.gov') }
```

Comment:
As this variable is only used in one of your specs, it should be moved to that context.